highlight/src/lib.rs

```diff
@@ -321,6 +381,14 @@ impl HighlightConfiguration {
                 best_index.map(Highlight)
             }));
     }
+
+    pub fn nonconformant_capture_names(&self) -> Vec<&String> {
+        return self
+            .names()
+            .iter()
+            .filter(|&n| !STANDARD_CAPTURE_NAMES.contains(n.as_str()))
```

Comment:
What do you think about also filtering out `n.starts_with('_')`? I've seen this convention used in a few languages with more complex queries to be able to use a value in a `#eq?`/`#match?` predicate without attaching a "real" highlight, like https://github.com/elixir-lang/tree-sitter-elixir/blob/7be390548a870ca9cb1bd7f59ac92457bbec7bf5/queries/highlights.scm#L93-L97

(Across all Helix queries: https://github.com/search?q=repo%3Ahelix-editor%2Fhelix+%2F%40_%5Cw%2B%2F+path%3Aruntime%2Fqueries&type=code)

Reply:
Yes, captures starting with `_` are treated as "private" by nvim-treesitter's Capture Police as well.

Reply:
A great idea. Incorporated.